src/mesg.c

```diff
@@ -316,13 +346,11 @@ send_priv(char *err, char *mesg) {
 }
 
 static int
-send_raw(char *err, char *mesg)
```

Comment:
I think keeping /RAW is a good idea, it lets the user send arbitrary messages. I do like your idea of keeping a default list of message types with no special handling though. Can you change this to keep /RAW and just add a new handler named, say, send_default_cmd or something?